ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/panelcontainer/v1/panelcontainer/clientlibs/editor/js/panelcontainer.js

```diff
@@ -97,41 +98,53 @@
         }
     }
 
-    function handleWrapData(containerEditor) {
-        let bindRef = containerEditor.querySelector("input[name='./dataRef']");
-        let minInputField = containerEditor.querySelector("coral-numberinput[name='./minItems']");
-        let maxInputField = containerEditor.querySelector("coral-numberinput[name='./maxItems']");
 
-        function isRepeatable() {
-            // TODO: to update after repeatability approach gets finalised
-            //return minInputField.value.length > 0 && maxInputField.value.length > 0;
-            return false;
+    function handleRepeat(dialogElement) {
+        const isRepeatable = function() {
+                let repeat = dialogElement.querySelector("input[name='./repeatable']");
```

Comment:
why are we using name? shouldn't we use BEM notations to select an element.

Reply:
since name is unique for a dialog we use this.